railties/lib/rails/commands/secrets/secrets_command.rb

```diff
@@ -17,10 +17,12 @@ def help
         end
       end
 
+      desc "setup", "Deprecated in favor of credentials. Run `bin/rails credentials:help`"
```

Comment:
Instead of having a period in the middle (but none at the end), we could use a dash:

```suggestion
      desc "setup", "Deprecated in favor of credentials -- run `bin/rails credentials:help`"
```